shard/src/guild_settings.py

```diff
@@ -66,7 +66,7 @@ def pug_emoji(self, new_emoji: str):
 
     @property
     def pug_timeout_speed(self) -> int:
-        return self._settings_dict['pug_timeout_speed'] if 'pug_timeout_speed' in self._settings_dict else 5
+        return self._settings_dict['pug_timeout_speed'] if 'pug_timeout_speed' in self._settings_dict else 15
```

Comment:
since we no longer add time each react we need to increase this to a sensible number